### ISSUE TITLE
[BUG] script-tmdbhelper-recommendations.xml doesn't respect <defaultcontrol>

### ISSUE THREAD

**Issue by roidy** [jurialmunkey/plugin.video.themoviedb.helper#1186]
### Describe the bug

I'm calling recommendations and filling list 5000 with data:-

`Runscript(plugin.video.themoviedb.helper,recommendations=5000|info=cast|true|info,window_id=1191,tmdb_type=tv,query=$INFO[ListItem.Title],year=$INFO[ListItem.Year])`

But in my script-tmdbhelper-recommendations.xml I want the default control to be 4999:-

```
<window type="dialog">
    <defaultcontrol always="true">4999</defaultcontrol>
    <onload>SetFocus(4999)</onload>

    <controls>
            <control type="list" id="4999">
                ...... my list
            </control>

            <!-- 5000 - Cast -->
            <include content="PosterWidget">
                <param name="id">5000</param>
            </include>

    </controls>
</window>
```

But when script-tmdbhelper-recommendations.xml loads control 5000 always has focus.

### Steps To Reproduce

none

### Checklist

- [X] I have searched the issues and this bug has not been reported.
- [X] I have updated TMDbHelper to the most recent version.
- [X] I have described the bug and included steps to reproduce the issue
- [ ] I have uploaded my debug log.

**Comment by roidy** [comment]
After digging into the code it seems that this isn't a bug, focus is hard coded on to the first list control:- 
https://github.com/jurialmunkey/plugin.video.themoviedb.helper/blob/ed1e2d2e9c3d253e3130c890f5ce9c1aa542d9d9/resources/tmdbhelper/lib/window/recommendations.py#L124

Could we get the option to specify an alternative control to be focused on load, ie:-

`Runscript(plugin.video.themoviedb.helper,recommendations=5000|info=cast|true|info,window_id=1191,focus_id=4999,.........)`

**Comment by roidy** [comment]
Added in this pull request:-

https://github.com/jurialmunkey/plugin.video.themoviedb.helper/pull/1187

**Comment by jurialmunkey** [comment]
Resolved by #1187
